cv32/sim/uvmt_cv32/xrun.mk

```diff
@@ -76,12 +76,20 @@ comp: mk_xrun_dir $(CV32E40P_PKG) $(OVP_MODEL_DPI)
 		-elaborate
 #		$(XRUN_VELABCOVERAGE)
 
-XRUN_COMP_RUN = $(XRUN_COMP_ARGS) $(XRUN_RUN_BASE_FLAGS)
```

Comment:
I believe this line is redundant since XRUN_COMP_RUN is assigned to either way in the next if/else block.